presentation/common/.gitignore

```diff
@@ -0,0 +1 @@
+en-PSEUDO/
```

Comment:
This is covered by the `.gitignore` at the root level. However, I believe the pseudolized files should be put straight into output folders, so this shouldn't be necessary at all.